cmd/nerd/main.go

```diff
@@ -47,7 +47,7 @@ func main() {
 	api, err := api.New(tServ, *conf)
 	if err != nil {
 		logger.Error("Error encountered initializing API", err)
-		return
+		os.Exit(1)
 	}
 	srv := &http.Server{
 		Addr:    ":5400",
```

Comment:
I think in line 69, you should add, after the logger.Error(...), an os.Exit(1)

Reply:
Fixed and pushed